valhalla/mjolnir/osmadmin.h

```diff
@@ -1,193 +0,0 @@
-#ifndef VALHALLA_MJOLNIR_OSMADMIN_H
```

Comment:
moved this into the other file. didnt make sense to have such heavily related things separate